packages/mgt-react/src/generated/spinner.ts

```diff
@@ -1,16 +1,13 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/consistent-type-definitions */
-import { registerMgtSpinnerComponent } from '@microsoft/mgt-components/dist/es6/components/components';
-import { ComponentMediaQuery } from '@microsoft/mgt-element';
+import { registerMgtSpinnerComponent } from '@microsoft/mgt-components';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import * as MicrosoftGraphBeta from '@microsoft/microsoft-graph-types-beta';
 import {wrapMgt} from '../Mgt';
 
-export type SpinnerProps = {
-	mediaQuery?: ComponentMediaQuery;
-}
+export type SpinnerProps = Record<string, never>
```

Comment:
Do we need it then?

Reply:
we need it, otherwise we'd need to change how the component is generated in a larger way. That said we don't need to export it.